src/aggregate/aggregate.h

```diff
@@ -123,20 +123,35 @@ typedef enum {
   QEXEC_S_ITERDONE = 0x02,
 } QEStateFlags;
 
-typedef struct AREQ {
+typedef struct AggregationPipeline {
   /* plan containing the logical sequence of steps */
   AGGPlan ap;
 
+  /** Context for iterating over the queries themselves */
+  QueryProcessingCtx qctx;
```

Comment:
I think it makes most sense to have only the `QueryProcessingCtx` and the `AGGPlan` in this new struct, since the separation is clearer than the current one and more natural for the other commands